### DISCUSSION TITLE
current status of Hylo (Dec 2024)

### DISCUSSION THREAD

**Discussion by notational** [hylo-lang/hylo#1630] [Q&A]
The [hylo-lang.org home page](https://www.hylo-lang.org) says “The current status of the project is described on our [roadmap page](https://www.hylo-lang.org/pages/implementation-status.html).”

This roadmap received its last substantive update in Feb 2023, and the page describes a series of features that were intended to be delivered after that date.

Has the status of the Hylo project changed since Feb 2023, or does that page (with its list of yet to be delivered features) still reflect the current state of development?

The roadmap page promises an “alpha version of Hylo” or “an experimental compiler” some time in the future. Does this exist yet?

**Comment by avillega** [comment]
I was about to make the same question! How close is Hylo of total world domination?

**Comment by kyouko-taiga** [comment]
Thanks for raising this issue, both of you. This roadmap page is really out of sync with our developments. We are behind on many points and ahead on others. But total domination is still on the menu, of course.

I pushed a time slot to make updates near the top of my stack.
